### PR TITLE
Add jquery-dotimeout@1.0 with git auto-update

### PR DESCRIPTION
**Pull request by LeaYeh**
**git repo url:** https://github.com/cowboy/jquery-dotimeout/tree/v1.0
**Watch 17
 Star 275
 Fork 30**
@Amomo   could you help me check out the PR for #6205   ? thank you~
- [x] Not found on cdnjs repo
- [x] No already exist issue and PR
- [x] Notable popularity 
- [x] Project has public repository on famous online hosting platform (or been hosted on npm) 
- [x] Has valid tags for each versions (for git auto-update)
- [x] Auto-update setup
- [x] Auto-update target/source is valid.
- [x] Auto-update filemap is correct

### CODE REVIEW

File: ajax/libs/jquery-dotimeout/package.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+  "filename": "jquery.ba-dotimeout.min.js",
+  "name": "jquery-dotimeout",
+  "version": "1.0",
+  "homepage": "https://github.com/cowboy/jquery-dotimeout",
+  "authors": [
```
Comment (PeterDaveHello) [inline ajax/libs/jquery-dotimeout/package.json]: Reason to use array here?

**Comment by LeaYeh** [comment]
my fault.
i fixed it!

**Comment by Amomo** [comment]
@LeaYeh I think it would be nicer to add description from README.md, like...
```
jQuery doTimeout takes the work out of delayed code execution, including interval and timeout management, polling loops and debouncing.
```

**Comment by LeaYeh** [comment]
@Amomo i add description field and rebase it, could you check this again?

**Comment by Amomo** [comment]
@PeterDaveHello I think this PR is fine, could you take a look? Thanks! :-D

**Pull request merged**